### ISSUE TITLE
Can not auto sync from server to database

### ISSUE THREAD

**Issue by JaidenW** [PotatoSauceVFX/Whitelist-Sync-2#18]
Hey, so i'm trying to set up this plugin to act as an auth server, basically. 
I have another plugin which synchronises roles from discord to the whitelist of the server, i want to use this plugin to then sync that whitelist to the SQL database **which i can do** With the /wl copyServerToDatabase command, However, i can only execute that command as a player in game.

What i would like to do, is use another plugin to schedule the use of that command every say, minute or so, or possibly react to when the discord bot updates the whitelist

The issue is, i can only execute that command as a player, i can not seem to execute it from console, it tells me i don't have permission to use that command from console or command block.

**Comment by JaidenW** [comment]
Hey, just wondering if you've made any progress on enabling this feature? It would be a fantastic utility for me, i'm an admin of a group which hosts random modpacks & other game servers, led by a streamer & we've lately been getting an increased amount of new members & currently are just relying on a discord channel which is only visible to ranked users, which just exposes the ips of the unwhitelisted servers, & we pretty much just manually make sure everyone is subbed, but as things are going, that's getting more and more difficult. I don't mean to rush you or anything, I understand how difficult it can be to find the time to give away to free projects like this, I've mentioned this to other people in similar positions to me, and basically everyone i've told has the same response, "Oh wow.... .... That's awesome.. could you set something up like that for my discord too?" I'm frankly a bit surprised someone hasn't built an "All In One" method of doing exactly what i'm trying to do, but i haven't found any aio solutions that work for forge as well, when testing with the command manually in game, everything seems to work as expected.

One thing i thought of after, since it seems like you've made it so the config does automatic syncing and commands are for manual syncing, maybe to keep it consistent, just add toggles for sync to server and sync from server and the frequency for each.

**Comment by PotatoSauceVFX** [comment]
@JaidenW I have been very busy recently and have not gotten a chance to look into this. Just so we are clear, the issue here is that you can't execute commands in the server console, right? Only as a player?

**Comment by PotatoSauceVFX** [comment]
@JaidenW Yes, please post it to the other discord. I am still not quite understanding what the issue is.
